tests/unit_tests/data_science_utils/schema/test_identifier.py

```diff
@@ -5,6 +5,7 @@
 
 
 class TestSchemaElement(ExasolIdentifierImpl):
+    __test__ = False
```

Comment:
Mark helper class with class name prefixed by `Test...` to not contain actual test cases. 
This avoids pytest warning about skipped tests due to `__init__` constructor, see ticket #183.